### PR TITLE
Adding Huber et al 2017 DFE for HomSap

### PR DESCRIPTION
**Pull request by chriscrsmith**
Adding a DFE for HomSap, from Huber et al., 2017. 

Related to #1096

### CODE REVIEW

File: stdpopsim/catalog/HomSap/dfes.py [hunk 1]
```diff
@@ -45,3 +45,53 @@ def _KimDFE():
...
+        distribution_type="g",  # gamma distribution
+        distribution_args=[gamma_mean, gamma_shape],
+    )    
+
+    return stdpopsim.DFE(
+        id=id,
+        description=description,
+        long_description=long_description,
+        mutation_types=[neutral, negative],
+        proportions=[0.3, 0.7], # [0.3 and 0.7 were used in Xin's analysis, but I couldn't find these values in the Huber paper]
```
Comment (andrewkern) [inline stdpopsim/catalog/HomSap/dfes.py:84]: i'm guess this line is too long for the pre-commit syntax checkers

Comment (chriscrsmith) [reply stdpopsim/catalog/HomSap/dfes.py]: ok, I wrapped that line and passed the pre-commit checks locally

**Review by andrewkern — APPROVED** [review]
looks good to me @chriscrsmith.

**Comment by andrewkern** [comment]
PR closes #1096

**Pull request merged**